AudioSystem/SequenceAudioConfig.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace DUCK.AudioSystem
+{
+	[CreateAssetMenu(menuName = "Sequence Audio Config (Fancy™)", order = 220)] // Right after Audio Mixer
+	public class AudioConfigSequence : AbstractAudioConfig
```

Comment:
I saw you changed the class name.

I think it could be either `SequencedAudioConfig` (like `SequencedAnimation`) or this `AudioConfigSequence`.

The thing is, you might want to change the file name too ;)